diff --git a/pyproject.toml b/pyproject.toml
index 6a2d850..d4e32dc 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -95,7 +95,7 @@ maintenance = ["conda-lock==3.0.1"]
 build-backend = "mesonpy"
 # Minimum requirements for the build system to execute.
 requires = [
-    "meson-python>=0.17.1,<0.19.0",
+    "meson-python>=0.17.1,<0.20.0",
     "cython>=3.1.2,<3.3.0",
     "numpy>=2,<2.4.0",
     "scipy>=1.10.0,<1.17.0",
