--- a/.pytest.ini 2025-01-24 13:15:05.000000000 +0000
+++ b/.pytest.ini 2026-01-13 14:16:29.155546796 +0000
@@ -7,9 +7,6 @@
     # ' <- This apostroph is just to fix syntax highlighting
 addopts =
     --import-mode=importlib
-    --no-cov-on-fail
-    --cov=semver
-    --cov-report=term-missing
     --doctest-glob='*.rst'
     --doctest-modules
-    --doctest-report ndiff
\ No newline at end of file
+    --doctest-report ndiff
