diff --git a/etc/Makefile.am b/etc/Makefile.am
index e3037a9..0d5b64c 100644
--- a/etc/Makefile.am
+++ b/etc/Makefile.am
@@ -4,16 +4,6 @@
 
 AUTOMAKE_OPTIONS = 1.7 gnits
 
-EXTRA_DIST = README locale netgroup netmasks securenets timezone ypserv.conf \
-		${MANS} ${XMLS}
+EXTRA_DIST = README locale netgroup netmasks securenets timezone ypserv.conf
 
 CLEANFILES = *~
-
-man_MANS = netgroup.5 ypserv.conf.5
-XMLS = netgroup.5.xml ypserv.conf.5.xml
-
-if ENABLE_REGENERATE_MAN
-%.5: %.5.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
diff --git a/makedbm/Makefile.am b/makedbm/Makefile.am
index 906bef6..56926d0 100644
--- a/makedbm/Makefile.am
+++ b/makedbm/Makefile.am
@@ -13,19 +13,8 @@ AM_CPPFLAGS = -I$(top_srcdir)/lib -I$(top_srcdir) -I$(top_builddir) \
 
 CLEANFILES = *~
 
-EXTRA_DIST = ${MANS} ${XMLS}
-
-man_MANS = makedbm.8
-XMLS = makedbm.8.xml
-
 libexec_PROGRAMS = makedbm
 
 makedbm_SOURCES = makedbm.c
 
 makedbm_LDADD =  @LIBDBM@  $(top_builddir)/lib/libyp.a @TIRPC_LIBS@
-
-if ENABLE_REGENERATE_MAN
-%.8: %.8.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
diff --git a/mknetid/Makefile.am b/mknetid/Makefile.am
index 4f63dd8..a4d91cb 100644
--- a/mknetid/Makefile.am
+++ b/mknetid/Makefile.am
@@ -13,11 +13,6 @@ AM_CPPFLAGS = -I$(top_srcdir)/lib -I$(top_srcdir) -I$(top_builddir) \
 
 CLEANFILES = *~
 
-EXTRA_DIST = ${MANS} ${XMLS}
-
-man_MANS = mknetid.8
-XMLS = mknetid.8.xml
-
 libexec_PROGRAMS = mknetid
 
 noinst_HEADERS = mknetid.h
@@ -26,9 +21,3 @@ mknetid_SOURCES = mknetid.c netid_hash.c
 
 mknetid_LDADD =  $(top_builddir)/lib/libyp.a
 
-if ENABLE_REGENERATE_MAN
-%.8: %.8.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
-
diff --git a/revnetgroup/Makefile.am b/revnetgroup/Makefile.am
index b31f4e2..44d5dcc 100644
--- a/revnetgroup/Makefile.am
+++ b/revnetgroup/Makefile.am
@@ -13,11 +13,6 @@ AM_CPPFLAGS = -I$(top_srcdir)/lib -I$(top_srcdir) -I$(top_builddir) \
 
 CLEANFILES = *~
 
-EXTRA_DIST = ${MANS} ${XMLS}
-
-man_MANS = revnetgroup.8
-XMLS = revnetgroup.8.xml
-
 libexec_PROGRAMS = revnetgroup
 
 noinst_HEADERS = hash.h
@@ -25,9 +20,3 @@ noinst_HEADERS = hash.h
 revnetgroup_SOURCES = revnetgroup.c getnetgrent.c hash.c
 
 revnetgroup_LDADD =  $(top_builddir)/lib/libyp.a
-
-if ENABLE_REGENERATE_MAN
-%.8: %.8.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
diff --git a/rpc.yppasswdd/Makefile.am b/rpc.yppasswdd/Makefile.am
index 7e2b3be..aa4dd62 100644
--- a/rpc.yppasswdd/Makefile.am
+++ b/rpc.yppasswdd/Makefile.am
@@ -13,11 +13,6 @@ AM_CPPFLAGS = -I$(top_srcdir)/lib -I$(top_srcdir) -I$(top_builddir) -I$(srcdir)
 
 CLEANFILES = *~
 
-EXTRA_DIST = ${MANS} ${XMLS}
-
-man_MANS = yppasswdd.8 rpc.yppasswdd.8
-XMLS = rpc.yppasswdd.8.xml
-
 noinst_HEADERS = yppwd_local.h yppasswd.x
 
 sbin_PROGRAMS = rpc.yppasswdd
@@ -27,9 +22,3 @@ rpc_yppasswdd_SOURCES = update.c yppasswd_xdr.c yppasswdd.c
 rpc_yppasswdd_LDADD =  @PIE_LDFLAGS@ $(top_builddir)/lib/libyp.a $(LIBDBM) $(LIBCRYPT) @SYSTEMD_LIBS@ @NSL_LIBS@ @TIRPC_LIBS@
 rpc_yppasswdd_CFLAGS = @PIE_CFLAGS@ @SYSTEMD_CFLAGS@ @NSL_CFLAGS@ @TIRPC_CFLAGS@
 
-if ENABLE_REGENERATE_MAN
-%.8: %.8.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
-
diff --git a/rpc.ypxfrd/Makefile.am b/rpc.ypxfrd/Makefile.am
index 1f54d92..caa31b9 100644
--- a/rpc.ypxfrd/Makefile.am
+++ b/rpc.ypxfrd/Makefile.am
@@ -13,11 +13,6 @@ AM_CPPFLAGS = -I$(top_srcdir)/lib -I$(top_srcdir) -I$(top_builddir) -I$(srcdir)
 
 CLEANFILES = *~
 
-EXTRA_DIST = ${MANS} ${XMLS}
-
-man_MANS = rpc.ypxfrd.8 ypxfrd.8
-XMLS = rpc.ypxfrd.8.xml
-
 sbin_PROGRAMS = rpc.ypxfrd
 
 rpc_ypxfrd_SOURCES = ypxfrd.c ypxfrd_server.c ypxfrd_svc.c
@@ -25,9 +20,3 @@ rpc_ypxfrd_SOURCES = ypxfrd.c ypxfrd_server.c ypxfrd_svc.c
 rpc_ypxfrd_LDADD = @PIE_LDFLAGS@ $(top_builddir)/lib/libyp.a \
 	@LIBDBM@ @SYSTEMD_LIBS@ @NSL_LIBS@ @TIRPC_LIBS@
 rpc_ypxfrd_CFLAGS = @PIE_CFLAGS@ @NSL_CFLAGS@ @SYSTEMD_CFLAGS@ @TIRPC_CFLAGS@
-
-if ENABLE_REGENERATE_MAN
-%.8: %.8.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
diff --git a/scripts/Makefile.am b/scripts/Makefile.am
index 9624008..5c2da95 100644
--- a/scripts/Makefile.am
+++ b/scripts/Makefile.am
@@ -6,13 +6,8 @@ AUTOMAKE_OPTIONS = 1.7 gnits
 
 CLEANFILES = *~ ${SCRIPTS}
 
-EXTRA_DIST = ${MANS} ${XMLS}
-
 varypdir = @YPMAPDIR@
 
-man_MANS = pwupdate.8 ypinit.8
-XMLS = pwupdate.8.xml ypinit.8.xml
-
 libexec_SCRIPTS = ypxfr_1perhour ypxfr_1perday ypxfr_2perday \
 		create_printcap match_printcap pwupdate ypinit
 
@@ -21,9 +16,3 @@ noinst_SCRIPTS = ypMakefile
 install-data-local:
 	$(mkdir_p) "$(DESTDIR)$(varypdir)"
 	$(install_sh_DATA) $(top_builddir)/scripts/ypMakefile "$(DESTDIR)$(varypdir)/Makefile"
-
-if ENABLE_REGENERATE_MAN
-%.8: %.8.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
diff --git a/yphelper/Makefile.am b/yphelper/Makefile.am
index 9c34cbe..a2021da 100644
--- a/yphelper/Makefile.am
+++ b/yphelper/Makefile.am
@@ -13,20 +13,9 @@ DEFS = @DEFS@ -DLOCALEDIR=\"$(localedir)\" -DYPBINDIR=\"$(libexecdir)\" \
 
 CLEANFILES = *~
 
-EXTRA_DIST = ${MANS} ${XMLS}
-
-XMLS = yphelper.8.xml
-man_MANS = yphelper.8
-
 libexec_PROGRAMS = yphelper
 
 yphelper_SOURCES = yphelper.c
 
 yphelper_LDADD = $(top_builddir)/lib/libyp.a @LIBDBM@ @NSL_LIBS@ @TIRPC_LIBS@
 yphelper_CFLAGS = @NSL_CFLAGS@ @TIRPC_CFLAGS@
-
-if ENABLE_REGENERATE_MAN
-%.8: %.8.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
diff --git a/yppush/Makefile.am b/yppush/Makefile.am
index 08057c8..41bcae4 100644
--- a/yppush/Makefile.am
+++ b/yppush/Makefile.am
@@ -10,12 +10,7 @@ DEFS = @DEFS@ -DLOCALEDIR=\"$(localedir)\" -DYPBINDIR=\"$(libexecdir)\" \
 	-DUSE_FQDN=@USE_FQDN@
 AM_CPPFLAGS = -I$(top_srcdir)/lib -I$(top_srcdir) -I$(top_builddir) -I$(srcdir)
 
-CLEANFILES = *~ ${MANS}
-
-EXTRA_DIST = ${MANS} ${XMLS}
-
-man_MANS = yppush.8
-XMLS = yppush.8.xml
+CLEANFILES = *~
 
 sbin_PROGRAMS = yppush
 
@@ -24,9 +19,3 @@ yppush_SOURCES = yppush.c
 yppush_LDADD =  @PIE_LDFLAGS@ @LIBDBM@ $(top_builddir)/lib/libyp.a \
 	@NSL_LIBS@ @TIRPC_LIBS@
 yppush_CFLAGS = @PIE_CFLAGS@ @NSL_CFLAGS@ @TIRPC_CFLAGS@
-
-if ENABLE_REGENERATE_MAN
-%.8: %.8.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
diff --git a/ypserv/Makefile.am b/ypserv/Makefile.am
index 500fe9e..20b274a 100644
--- a/ypserv/Makefile.am
+++ b/ypserv/Makefile.am
@@ -11,19 +11,8 @@ AM_CPPFLAGS = -I$(top_srcdir)/lib -I$(top_srcdir) -I$(top_builddir) -I$(srcdir)
 
 CLEANFILES = *~
 
-EXTRA_DIST = ${MANS} ${XMLS}
-
-man_MANS = ypserv.8
-XMLS = ypserv.8.xml
-
 sbin_PROGRAMS = ypserv
 
 ypserv_SOURCES = ypserv.c server.c ypserv_xdr.c
 ypserv_CFLAGS = @PIE_CFLAGS@ @NSL_CFLAGS@ @SYSTEMD_CFLAGS@ @TIRPC_CFLAGS@
 ypserv_LDADD =  @PIE_LDFLAGS@ ../lib/libyp.a @NSL_LIBS@ @LIBDBM@ @SYSTEMD_LIBS@ @TIRPC_LIBS@
-
-if ENABLE_REGENERATE_MAN
-%.8: %.8.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
diff --git a/ypxfr/Makefile.am b/ypxfr/Makefile.am
index 7b01f52..fe4dd49 100644
--- a/ypxfr/Makefile.am
+++ b/ypxfr/Makefile.am
@@ -10,12 +10,6 @@ AM_CPPFLAGS = -I$(top_srcdir)/lib -I$(top_srcdir) -I$(top_builddir) -I$(srcdir)
 
 CLEANFILES = *~
 
-
-EXTRA_DIST = ${MANS} ${XMLS}
-
-man_MANS = ypxfr.8
-XMLS = ypxfr.8.xml
-
 noinst_HEADERS = ypxfr.h
 
 libexec_PROGRAMS = ypxfr
@@ -24,9 +18,3 @@ ypxfr_SOURCES = ypxfr.c ypxfr_clnt.c ypxfr_xdr.c
 
 ypxfr_LDADD = @LIBDBM@ $(top_builddir)/lib/libyp.a @NSL_LIBS@ @TIRPC_LIBS@
 ypxfr_CFLAGS = @NSL_CFLAGS@ @TIRPC_CFLAGS@
-
-if ENABLE_REGENERATE_MAN
-%.8: %.8.xml
-	$(XMLLINT) --nonet --xinclude --postvalid --noout $<
-	$(XSLTPROC) -o $(srcdir)/$@ --path $(srcdir) --xinclude --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl $<
-endif
