diff --git a/src/api/python/speechd_config/Makefile.in b/src/api/python/speechd_config/Makefile.in
index 6ae96a0..8999047 100644
--- a/src/api/python/speechd_config/Makefile.in
+++ b/src/api/python/speechd_config/Makefile.in
@@ -425,7 +425,7 @@ buildconfig_edit = sed \
 
 EXTRA_DIST = buildconfig.py.in speechd.desktop.in
 @HAVE_HELP2MAN_TRUE@@HAVE_PYTHON_TRUE@dist_man1_MANS = \
-@HAVE_HELP2MAN_TRUE@@HAVE_PYTHON_TRUE@	spd-conf.1
+@HAVE_HELP2MAN_TRUE@@HAVE_PYTHON_TRUE@
 
 all: all-am
 
@@ -856,7 +856,7 @@ $(abs_builddir)/config.py:
 
 buildconfig.py: $(srcdir)/buildconfig.py.in
 
-@HAVE_HELP2MAN_TRUE@@HAVE_PYTHON_TRUE@spd-conf.1: buildconfig.py $(abs_builddir)/__init__.py $(abs_builddir)/config.py
+@HAVE_HELP2MAN_TRUE@@HAVE_PYTHON_TRUE@: buildconfig.py $(abs_builddir)/__init__.py $(abs_builddir)/config.py
 @HAVE_HELP2MAN_TRUE@@HAVE_PYTHON_TRUE@	LC_ALL=C PYTHONPATH=$(builddir)/.. PYTHONDONTWRITEBYTECODE=1 help2man -N -n "configure Speech Dispatcher and diagnose problems" --output=$@ $(srcdir)/spd-conf
 
 -include $(top_srcdir)/git.mk
